pkg/openstack/dataplane.go

```diff
@@ -0,0 +1,64 @@
+package openstack
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openstack-k8s-operators/lib-common/modules/common/condition"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/helper"
+
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	dataplanev1beta1 "github.com/openstack-k8s-operators/dataplane-operator/api/v1beta1"
+	corev1beta1 "github.com/openstack-k8s-operators/openstack-operator/apis/core/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+// ReconcileDataplaneNode -
+func ReconcileDataplaneNode(ctx context.Context, instance *corev1beta1.OpenStackControlPlane, helper *helper.Helper) (ctrl.Result, error) {
+	if !instance.Spec.DataPlaneNode.Enabled {
+		return ctrl.Result{}, nil
+	}
+
+	dataPlaneNode := &dataplanev1beta1.OpenStackDataPlaneNode{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "dataplanenode", //FIXME
+			Namespace: instance.Namespace,
+		},
+	}
```

Comment:
I'm not so sure if I need to add more initial values from: https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenode_deployment.yaml

@slagle wdyt?

Reply:
within reason I would add what is needed here to streamline a demo of this workflow for now. If it really only applies to our Q1 demo and we might have better ways of setting defaults in the future perhaps comment them to address those concerns. If we run the demo via the OpenShift web console I would also add that the default config/sample for each CR is going to get slurped into alm-examples for the CSV and would thus get applied as default values in the UI there. But that only applies in the OpenShift web console I think